planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_out/pull_out_parameters.hpp

```diff
@@ -38,6 +38,7 @@ struct PullOutParameters
   double deceleration_interval;
   // geometric pull out
   bool enable_geometric_pull_out;
+  bool enable_combined_path;
```

Comment:
This parameter name is a little confusing.
(path is usually combined. )

Reply:
I think a name like ```divide_pull_out_path``` is easier to understand

Reply:
thanks, updated the name in https://github.com/autowarefoundation/autoware.universe/pull/2669/commits/18ce9aff21cf416db9edd1495528f53aa82c481c